### PR TITLE
Update netdata-charts.md

### PR DESCRIPTION
**Pull request by Ancairon**
##### Summary

It was brought up internally that we have not documented this feature, so I added updated screenshots and descriptions

### CODE REVIEW

File: docs/dashboards-and-charts/netdata-charts.md [hunk 1]
```diff
@@ -100,22 +100,27 @@ names:
...
+
+- **Manage Alerts**: manage [Alert configurations](/docs/dashboards-and-charts/alerts-tab.md#alert-configurations-tab) for this chart.
 - **Chart info**: get more information relevant to the chart you are interacting with.
 - **Chart type**: change the chart type from **line**, **stacked**, **area**, **stacked bar** and **multi bar**.
 - **Enter fullscreen mode**: expand the current chart to the full size of your screen.
-- **Add chart to dashboard**: add the chart to an existing custom dashboard or directly create a new one that includes the chart.
+- **User settings**: save your settings for the chart at hand, so it persists across dashboard reloads.
+  - Personal has the top priority.
+  - Room and Space wide settings for a chart are shared across all users that don't have personal setting for it. 
+- **Drag and Drop the chart to a Dashboard**: add the chart to an existing custom [Dashboard](/docs/dashboards-and-charts/dashboards-tab.md) or directly create a new one that includes the chart.
```
Comment (ilyam8) [inline docs/dashboards-and-charts/netdata-charts.md:123]:
```suggestion
- **User settings**: save your settings for the chart at hand, so it persists across dashboard reloads.
  - Personal has the top priority.
  - Room and Space settings for a chart are shared across all users who don't have personal settings for it. 
- **Drag and Drop the chart to a Dashboard**: add the chart to an existing custom [Dashboard](/docs/dashboards-and-charts/dashboards-tab.md) or directly create a new one that includes the chart.
```

File: docs/dashboards-and-charts/netdata-charts.md [hunk 2]
```diff
@@ -100,22 +100,27 @@ names:
 
 When you start interacting with a chart, you'll notice valuable information on the top bar:
 
-<img src="https://user-images.githubusercontent.com/70198089/236133832-fad45e65-5bd6-4fd1-8d68-33acf69fff5c.png" width="900"/>
+<img src="https://github.com/netdata/netdata/assets/70198089/75d700de-bc7d-4b96-b73d-7b248b83afea" width="900"/>
 
-The elements that you can find on this top bar are:
+The elements that you can find on this title bar are:
```
Comment (ilyam8) [inline docs/dashboards-and-charts/netdata-charts.md:105]:
```suggestion
Top bar elements:
```

We call it the top bar (the sentence above), why "title" bar here

Comment (Ancairon) [reply docs/dashboards-and-charts/netdata-charts.md:105]: Let me check!

Comment (Ancairon) [reply docs/dashboards-and-charts/netdata-charts.md:105]: I will replace top bar above, sounds better as "title bar"

File: docs/dashboards-and-charts/netdata-charts.md [hunk 3]
```diff
@@ -100,22 +100,27 @@ names:
 
 When you start interacting with a chart, you'll notice valuable information on the top bar:
```
Comment (Ancairon) [inline docs/dashboards-and-charts/netdata-charts.md:101]:
```suggestion
When you start interacting with a chart, you'll notice valuable information on the title bar:
```

**Review by ilyam8 — DISMISSED** [review]

File: docs/dashboards-and-charts/netdata-charts.md [hunk 4]
```diff
@@ -98,24 +98,29 @@ names:
...
 ## Title bar
 
-When you start interacting with a chart, you'll notice valuable information on the top bar:
+When you start interacting with a chart, you'll notice valuable information on the title bar:
 
-<img src="https://user-images.githubusercontent.com/70198089/236133832-fad45e65-5bd6-4fd1-8d68-33acf69fff5c.png" width="900"/>
+<img src="https://github.com/netdata/netdata/assets/70198089/75d700de-bc7d-4b96-b73d-7b248b83afea" width="900"/>
 
-The elements that you can find on this top bar are:
+The elements that you can find on this title bar are:
```
Comment (Ancairon) [inline docs/dashboards-and-charts/netdata-charts.md:105]:
```suggestion
The Title bar elements are:
```

Comment (Ancairon) [inline docs/dashboards-and-charts/netdata-charts.md:105]:
```suggestion
The elements that you can find on this Title bar are:
```

File: docs/dashboards-and-charts/netdata-charts.md [hunk 5]
```diff
@@ -98,24 +98,29 @@ names:
 
 ## Title bar
 
-When you start interacting with a chart, you'll notice valuable information on the top bar:
+When you start interacting with a chart, you'll notice valuable information on the title bar:
```
Comment (Ancairon) [inline docs/dashboards-and-charts/netdata-charts.md:101]:
```suggestion
When you start interacting with a chart, you'll notice valuable information on the Title bar:
```

File: docs/dashboards-and-charts/netdata-charts.md [hunk 6]
```diff
@@ -98,24 +98,29 @@ names:
...
 ## Title bar
 
-When you start interacting with a chart, you'll notice valuable information on the top bar:
+When you start interacting with a chart, you'll notice valuable information on the Title bar:
 
-<img src="https://user-images.githubusercontent.com/70198089/236133832-fad45e65-5bd6-4fd1-8d68-33acf69fff5c.png" width="900"/>
+<img src="https://github.com/netdata/netdata/assets/70198089/75d700de-bc7d-4b96-b73d-7b248b83afea" width="900"/>
 
-The elements that you can find on this top bar are:
+The Title bar elements are:
```
Comment (ilyam8) [inline docs/dashboards-and-charts/netdata-charts.md:105]:
```suggestion
The Title bar elements:
```

**Review by ilyam8 — DISMISSED** [review]

File: docs/dashboards-and-charts/netdata-charts.md [hunk 7]
```diff
@@ -98,24 +98,29 @@ names:
...
 ## Title bar
 
-When you start interacting with a chart, you'll notice valuable information on the top bar:
+When you start interacting with a chart, you'll notice valuable information on the Title bar:
 
-<img src="https://user-images.githubusercontent.com/70198089/236133832-fad45e65-5bd6-4fd1-8d68-33acf69fff5c.png" width="900"/>
+<img src="https://github.com/netdata/netdata/assets/70198089/75d700de-bc7d-4b96-b73d-7b248b83afea" width="900"/>
 
-The elements that you can find on this top bar are:
+The Title bar elements:
```
Comment (Ancairon) [inline docs/dashboards-and-charts/netdata-charts.md:105]:
```suggestion
The Title bar elements are:
```

**Review by ilyam8 — APPROVED** [review]

**Pull request merged**